superset-frontend/src/explore/components/controls/VizTypeControl/VizTypeGallery.tsx

```diff
@@ -478,6 +478,7 @@ const Selector: React.FC<{
       key={selector}
       name={selector}
       className={cx(className, isSelected && 'selected')}
+      role="button"
```

Comment:
Since SelectorLabel is a button element, role is not necessary (implicitly defined) but type is necessary for a11y POV. 

```suggestion
      type="button"
```